stdlib/http/client.pyi

```diff
@@ -180,7 +185,7 @@ class HTTPConnection:
     def connect(self) -> None: ...
     def close(self) -> None: ...
     def putrequest(self, method: str, url: str, skip_host: bool = False, skip_accept_encoding: bool = False) -> None: ...
-    def putheader(self, header: str | bytes, *argument: str | bytes) -> None: ...
+    def putheader(self, header: str | bytes, *argument: _HeaderValue) -> None: ...
```

Comment:
Nit: While it doesn't make any practical difference, the argument is called `values` in the source, so we should use that, too:

```suggestion
    def putheader(self, header: str | bytes, *values: _HeaderValue) -> None: ...
```

Reply:
done, thanks!